### DISCUSSION TITLE
Is it possible to include notes/comment about the history changes?

### DISCUSSION THREAD

**Discussion by RaylynYu** [requarks/wiki#4309] [Help / Questions]
Hello, 

I was wondering if there is a way to add notes or comments to a page when it's being edited. That way when you or another wiki user looks at the history they can see what changes where made or why they were made by reading the notes/comments. 

OS: CentOS 7
Node.js: 14.16.1 
Wiki.js version: 2.5.201
Database engine: PostgreSQL 13.2

Thank you, 
Ray

**Comment by nunick** [comment]
I was wondering this as well. Can anything confirm if this is a feature?

**Comment by Imberflur** [comment]
Hi, I just had the same question. So far, I haven't seen anything that indicates this feature exists though.
